superset/models.py

```diff
@@ -149,13 +149,19 @@ def changed_by_fk(cls):  # noqa
             Integer, ForeignKey('ab_user.id'),
             default=cls.get_user_id, onupdate=cls.get_user_id, nullable=True)
 
-    @renders('created_on')
+    def _user_link(self, user):
+        if not user:
+            return ''
+        url = '/superset/profile/{}/'.format(user.username)
+        return '<a href="{}">{}</a>'.format(url, escape(user) or '')
```

Comment:
that's for the `list_columns` view of the CRUD generated "Dashboards" and "Slices" page. We're tricking the list view into displaying links.